src/features/editable-table/index.tsx

```diff
@@ -0,0 +1,146 @@
+import { localStorageController } from '@/shared/utils';
+import { VariablesData } from '@/types/types';
+import { Button, Empty, Flex, Form, Popconfirm, Space, Table, TableProps, Typography } from 'antd';
```

Comment:
```suggestion
import { Button, Empty, Flex, Form, Table, TableProps, Typography } from 'antd';
```

Those components were not used. Unless you plan to add them, they can be deleted from import.